docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -17,7 +17,7 @@ wp.blocks.registerBlockStyle( 'core/quote', {
 
 The example above registers a block style variation named `fancy-quote` to the `core/quote` block. When the user selects this block style variation from the styles selector, an `is-style-fancy-quote` className will be added to the block's wrapper.
 
-By adding `isDefault: true`, you can make registered style variation to be active by default when a block is inserted.
+By adding `isDefault: true` you can make the registered style variation to be used as a fallback if no style in `registerBlockType()` has been set as the default.
```

Comment:
Maybe something like?

> By adding `isDefault: true` you can mark the registered style variation as the one that is active without any custom className. 

(Or something in that vein, it's hard to explain :P)